optuna/terminator/improvement/gp/botorch.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
 from typing import List
 from typing import Optional
 from typing import Tuple
```

Comment:
[nit] Might be good to replace all `typing.List` and `typing.Tuple` in this file with `list` and `tuple`. Do you want to work on it as a follow-up?

Reply:
Thanks fixed this in this PR.